tracing/tracer.py

```diff
@@ -134,46 +135,43 @@ def send_trace_probe(self, switch, in_port, probe_pkt):
             {switch & port}
         """
         timeout_control = 0  # Controls the timeout of 1 second and two tries
-
-        while True:
-            log.warning('Trace %s: Sending POut to switch: %s and in_port %s '
-                        % (self.id, switch.dpid, in_port))
+        while not self.trace_ended:
+            log.warning(f'Trace {self.id}: Sending POut to switch:'
+                        f' {switch.dpid} and in_port {in_port}.'
+                        f' Timeout: {self.init_entries.timeout}')
```

Comment:
Great to have `timeout` in the log. @Alopalao could you change the log level from warning to info? Originally, it ended up using a log level that's not coherent.